### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by kentonh**
Updates title of page and title meta element
Adds better explainer text in "About" section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,6 +73,13 @@ <h2 class="section-heading text-white">About</h2>
             <hr class="light">
             <p class="text">This is a blatant attempt to piggyback off the popularity of the global <a class="dark" href="https://hacktoberfest.digitalocean.com" target="_blank">Hacktoberfest event</a> from Digital Ocean and GitHub. Submit four pull requests to qualifying devICT projects during the month of October and receive a t-shirt!</p>
             <a class="btn btn-dark sr-button" href="#get-started">Learn More</a>
+            <p>Hacktoberfest is a great way for anyone to learn about open source, contributions to a team, and helps build a better resume and portfolio.<br>
+               This event was created to:
+            <ul>
+               <li>Bring awareness to the open source projects that make Wichita a better place to live</li>
+               <li>Bring light to the diversity of the Wichita Tech Commnity</li>
+               <li>Create opportunities for those individuals who are underrepresented in tech</li>
+            </ol>
```
Comment (jcbwlkr) [inline index.html]: Should be ul not ol

Comment (jcbwlkr) [reply index.html]: Add closing p tag?

**Comment by kentonh** [comment]
@jcbwlkr whoops, typing too fast...

**Comment by KatherineMichel** [comment]
Awesome. Thank you for the help! Sorry it took a while. GitHub did not notify me of the pull request in the browser like it normally does. Hope you don't mind, but I moved the button to the bottom of the section. I think it is more aesthetically pleasing. I want to make the button be smooth scroll and reduce the space in between the bullets and the text when I have the chance or these can be up-for-grabs issues. @kentonh @jcbwlkr

**Pull request merged**